authors/apps/articles/views/comments.py

```diff
@@ -267,6 +267,41 @@ def get_error_text_response(self, error, code):
         }
 
 
+class CommentHistoryView(generics.RetrieveAPIView):
+    queryset = CommentHistory.objects.all()
+
+    permission_classes = (IsAuthenticated,)
+
+    serializer_class = CommentSerializer
+
+    def get(self, request, *args, **kwargs):
+        # We need to get the comment id from the url parameter.
+        comment_id = kwargs["pk"]
+
+        # This tries to fetch a list of comment edits for the specified comment
+        # from the database. If the comment id provided does not match any in
+        # the database, an error 404 response is sent back to the API user.
+        comment_history = CommentHistory.objects.filter(comment=comment_id)
+
+        if not comment_history:
```

Comment:
It would be appropriate to use a render class to define the response and remove the similar blocks of code.

Reply:
I have added the renderer class. Thanks!